### PR TITLE
Added Euclidean Algorithm for lisp

### PR DESCRIPTION
**Pull request by Trashtalk217**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -57,3 +57,5 @@ NIFR91
 Michal Hanajik
 <br>
 Bendik Samseth
+Trashtalk 
+Trashtalk
```
Comment (berquist) [inline CONTRIBUTORS.md]: Since you've already done this as part of https://github.com/algorithm-archivists/algorithm-archive/pull/348, you'll need to undo these changes.

File: contents/euclidean_algorithm/code/lisp/euclidean_algorithm.lisp [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            (euclid_sub a (- b a)))))
+
+(defun euclid_mod (a b)
+    (if (zerop b)
+        a
+        (euclid_mod b (mod a b))))
+
+;; built-in funciton: (gcd 80 40)
+(print (euclid_sub 24 27))      ;should output 3
+(print (euclid_mod 90 27))      ;should output 9
```
[the remark this answers is not in the document]
Comment (jiegillet) [reply contents/euclidean_algorithm/code/lisp/euclidean_algorithm.lisp]: super minor: the comment misspells "funciton"
